ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -189,6 +197,11 @@ public void visitNote() {
         clickOnLast(VISIT_NOTE);
         waitForElement(DIAGNOSIS_SEARCH_CONTAINER);
     }
+    
+    public AllergyPage clickOnAllergyManagement() {
+        clickOn(ADD_ALLERGY);
+        return new AllergyPage(this);
+    }
```

Comment:
Make use of the existing method that can achive what clickOnAllergyManagement() method can accomplish. Have you had a chance to have a look on what [clickOnAllergiesWidgetLink() ](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/f2ba851ef03ca58c46fa514308d49a24ba05c017/ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java#L449) can do ?

Reply:
Thanks. This **clickOnAllergyManagement** comes from allergyPage and as you can see the element used are different., the one adds a new allergy and clickOnAllergiesWidgetLink()  is on patient dashboard

Reply:
Take a look at the element reference of [ALLERGIES_LINK](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/66bee8b4a374095aa6abad7b3644d4369b591bdb/ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java#L79) and [ADD_ALLERGY](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/a6b54f61adddfdddd6dc06fae916d5eaf4a2943f/ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java#L37)

WDYT ?

Reply:
Let me address this comment.